keyboards/primekb/meridian/meridian.c

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2020 Holten Campbell
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "meridian.h"
+
+//Initialize B12 for in-switch caps lock
+void keyboard_pre_init_user(void){
+    setPinOutput(B12);
+}
+
+//Initialize all RGB indicators to 'off'
+void keyboard_post_init_user(void) {
+    rgblight_setrgb_at(0, 0, 0, 0); // [..., 0] = top LED
+    rgblight_setrgb_at(0, 0, 0, 1); // [..., 1] = middle LED
+    rgblight_setrgb_at(0, 0, 0, 2); // [..., 2] = bottom LED
+}
```

Comment:
```suggestion
void keyboard_post_init_kb(void) {
    rgblight_setrgb_at(0, 0, 0, 0); // [..., 0] = top LED
    rgblight_setrgb_at(0, 0, 0, 1); // [..., 1] = middle LED
    rgblight_setrgb_at(0, 0, 0, 2); // [..., 2] = bottom LED

    keyboard_post_init_user();
}
```

or 

```suggestion
__attribute__((weak))
void keyboard_post_init_user(void) {
    rgblight_setrgb_at(0, 0, 0, 0); // [..., 0] = top LED
    rgblight_setrgb_at(0, 0, 0, 1); // [..., 1] = middle LED
    rgblight_setrgb_at(0, 0, 0, 2); // [..., 2] = bottom LED
}
```

Reply:
I had initially had all these functions using _kb but the indicators wouldn't work with via keymap unless I was using _user. __attribute__((weak)) seems to be what I was missing to make them work as intended with via.